config/webpack.config.js

```diff
@@ -197,17 +197,22 @@ const configGenerator = (buildOptions, apps) => {
         __API__: JSON.stringify(buildOptions.api),
       }),
 
-      new ExtractTextPlugin({
+      new MiniCssExtractPlugin({
         filename: !isOptimizedBuild
           ? '[name].css'
           : `[name].[contenthash]-${timestamp}.css`,
       }),
       new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+    ],
+  };
+
+  if (!buildOptions.watch) {
```

Comment:
Running this plugin when using the watch task now results in very slow rebuilds. Not sure why, but we don't need it for the watch task.